client/api_account.go

```diff
@@ -0,0 +1,160 @@
+package client
+
+import (
+	"context"
+
+	gnfdSdkTypes "github.com/bnb-chain/greenfield/sdk/types"
+	paymentTypes "github.com/bnb-chain/greenfield/x/payment/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authTypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	types3 "github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+type Account interface {
+	GetAccount(ctx context.Context, address string) (authTypes.AccountI, error)
+	GetAccountBalance(ctx context.Context, address string) (*sdk.Coin, error)
+	GetPaymentAccount(ctx context.Context, address string) (*paymentTypes.PaymentAccount, error)
+	GetModuleAccounts(ctx context.Context) ([]authTypes.ModuleAccountI, error)
+	GetModuleAccountByName(ctx context.Context, name string) (authTypes.ModuleAccountI, error)
+	CreatePaymentAccount(ctx context.Context, address string, txOption *gnfdSdkTypes.TxOption) (*sdk.TxResponse, error)
+	GetPaymentAccountsByOwner(ctx context.Context, owner string) ([]*paymentTypes.PaymentAccount, error)
+	Transfer(ctx context.Context, toAddress string, amount int64, txOption *gnfdSdkTypes.TxOption) (*sdk.TxResponse, error)
```

Comment:
Suggest to add blank lines to group different APIs, such as payment-related should be in a group.  And the APIs in the same group should follow the Get*** APIs behind the other APIs.